### PR TITLE
Update Vert.x to 4.5.4

### PR DESCRIPTION
**Pull request by fab-10**
### Thanks for sending a pull request! Have you done the following?

- [ ] Checked out our [contribution guidelines](https://github.com/hyperledger/besu/blob/main/CONTRIBUTING.md)?
- [ ] Considered documentation and added the `doc-change-required` label to this PR [if updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).
- [ ] Considered the changelog and included an [update if required](https://wiki.hyperledger.org/display/BESU/Changelog).
- [ ] For database changes (e.g. KeyValueSegmentIdentifier) considered compatibility and performed forwards and backwards compatibility tests

### Most advanced CI tests are deferred until PR approval, but you could:

- [ ] locally run all unit tests via: `./gradlew build`
- [ ] locally run all acceptance tests via: `./gradlew acceptanceTest`
- [ ] locally run all integration tests via: `./gradlew integrationTest`
- [ ] locally run all reference tests via: `./gradlew ethereum:referenceTests:referenceTests`

## PR description

A small change was required to update Vert.x, since in the new version the CORS check have changed, and if using `.*` to match any origin, the origin must have as protocol one of: `http`, `https` or `ftp`, but since we also want to support browser extensions as origins, which have protocols like `moz-extension://` , then the regex should be written has `.*://.*`  

[In the version websocket write handlers are disabled by default](https://github.com/eclipse-vertx/vert.x/issues/4625), and since we use them, we need to enable them.

### CODE REVIEW

File: ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java [hunk 1]
```diff
@@ -580,7 +580,7 @@ private String buildCorsRegexFromConfig() {
       return "";
     }
     if (config.getCorsAllowedDomains().contains("*")) {
-      return ".*";
+      return ".*://.*";
```
Comment (siladu) [inline ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java:583]: why was this required?

**Review by siladu — APPROVED** [review]
LGTM

**Comment by shemnon** [comment]
PR #6666?  I always knew there was something unseemly about vertx ;)

File: ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java (hunk 1 above)
Comment (fab-10) [reply ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java:583]: see the PR description
